internal/sweep/sweep.go

```diff
@@ -55,7 +55,7 @@ func SweepServices(region, t string) error {
 			continue
 		}
 
-		if !strings.Contains(s.Name, "test-acc-") {
```

Comment:
Can you also add the a K8s acc and e2e tests prefix?

Reply:
Done. But actually operators tests are mess. Naming may vary and doesn't have strict pattern. Can't we just sweep all?

Reply:
I think we can be on the same side if we use `k8s-*` for K8s tests, all of the tests start with this prefix